<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>359</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>359</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pyLottoSimu</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="widget_lottery_numbers">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>320</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>lottery numbers</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_big_number, QLabel#label_last_draw_number {	background-color: none}</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>478</width>
       <height>318</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="main_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plaintextedit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="output field">output field</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Lottozahlen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>lottery numbers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>351</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_numbers">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Numbers:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_random_numbers">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>random numbers</string>
     </property>
     <property name="statusTip">
      <string>random numbers</string>
     </property>
     <property name="text">
      <string>random numbers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clean_output_text">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>output field empty</string>
     </property>
     <property name="text">
      <string>output field empty</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_big_number">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>start draw</string>
     </property>
     <property name="statusTip">
      <string>start draw</string>
     </property>
     <property name="text">
      <string>start draw</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Speed</string>
     </property>
     <property name="statusTip">
      <string>Speed</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_last_draw_number">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>171</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rdbtn_show_draw_after">
        <property name="styleSheet">
         <string notr="true">color:black; background-color: whitesmoke;</string>
        </property>
        <property name="text">
         <string>show draw after</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_draw_overview">
        <property name="text">
         <string>show draw overview</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>label_Lottozahlen</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>btn_random_numbers</zorder>
    <zorder>clean_output_text</zorder>
    <zorder>btn_start</zorder>
    <zorder>horizontalSlider</zorder>
    <zorder>label_speed</zorder>
    <zorder>label_last_draw_number</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>plaintextedit</zorder>
    <zorder>label_big_number</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_info"/>
   </widget>
   <widget class="QMenu" name="menu_mode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="action_lotto_simulation"/>
    <addaction name="action_lotto_system"/>
    <addaction name="action_uniform_distribution"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_mode"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionZufallsgenerator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>random numbers</string>
   </property>
  </action>
  <action name="action_lotto_simulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lotto simulation</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_go_to_the_website">
   <property name="text">
    <string>Go to the website</string>
   </property>
  </action>
  <action name="action_lotto_system">
   <property name="text">
    <string>Lotto system</string>
   </property>
  </action>
  <action name="action_uniform_distribution">
   <property name="text">
    <string>Uniform Distribution</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
